website/content/en/docs/building-operators/helm/quickstart.md

```diff
@@ -3,321 +3,102 @@ title: Helm Based Operator Quickstart
-INFO[0000] operator-sdk Version: v0.2.0+git
-```
-
-### 3. Deploy your Operator with the Operator Lifecycle Manager (OLM)
-
-OLM will manage creation of most if not all resources required to run your operator,
-using a bit of setup from other `operator-sdk` commands. Check out the OLM integration
-[user guide][quickstart-bundle] for more information.
-
-## Deploy the Nginx custom resource
-
-Apply the nginx CR that we modified earlier:
-
-```sh
-$ kubectl apply -f config/samples/example_v1alpha1_nginx.yaml
-```
-
-Ensure that the nginx-operator creates the deployment for the CR:
-
-```sh
-$ kubectl get deployment
-NAME           READY   UP-TO-DATE   AVAILABLE   AGE
-nginx-sample   2/2     2            2           2m13s
 ```
 
-Check the pods to confirm 2 replicas were created:
-
-```sh
-$ kubectl get pods
-NAME                                                   READY   STATUS    RESTARTS   AGE
-nginx-sample-c786bfdcf-4g6md                           1/1     Running   0          81s
-nginx-sample-c786bfdcf-6bhmx                           1/1     Running   0          81s
+## Building and Pushing the Project Image
 
-Check that the service port is set to `8080`:
+To build and push your image to your repository :
 
 ```sh
-$ kubectl get service
-NAME                                      TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)    AGE
-nginx-sample                              ClusterIP   10.96.26.3   <none>        8080/TCP   1m
+$ make docker-build docker-push IMG=<some-registry>/<project-name>:tag
 ```
 
-### Update the replicaCount and remove the port
+**Note** To allow the cluster pull the image the repository needs to be set as public. 
 
-Change the `spec.replicaCount` field from 2 to 3, remove the `spec.service`
-field:
+## Running it on Cluster
 
-```sh
-$ cat config/samples/example_v1alpha1_nginx.yaml
-apiVersion: example.com/v1alpha1
-kind: Nginx
-metadata:
-  name: nginx-sample
-spec:
-  replicaCount: 3
-```
+Deploy the project to the cluster:
 
-And apply the change:
 ```sh
-$ kubectl apply -f config/samples/example_v1alpha1_nginx.yaml
+$ make deploy IMG=<some-registry>/<project-name>:tag
 ```
 
-Confirm that the operator changes the deployment size:
-
-```sh
-$ kubectl get deployment
-NAME                                           DESIRED   CURRENT   UP-TO-DATE     AGE
-nginx-sample                                   3/3       3            3           7m29s
-```
+## Applying the CR's into the cluster:
 
-Check that the service port is set to the default (`80`):
+To create instances (CR's) of the `Nginx` Kind (CRD) in the same namespaced of the operator 
 
 ```sh
-$ kubectl get service
-NAME                                      TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)  AGE
-nginx-sample                              ClusterIP   10.96.152.76    <none>        80/TCP   7m54s
+$ kubectl apply -f config/samples/demo_v1_nginx.yaml -n nginx-operator-system
 ```
 
-### Troubleshooting
+## Uninstall CRDs
```

Comment:
This section isn't required. `make undeploy` will delete the CRDs.

Also note that CRDs should be deleted after the operator is torn down. Otherwise the operator starts spamming the logs that it can't watch for instances of the (now non-existing) CRD.